bin/git_time_extractor

```diff
@@ -68,13 +70,28 @@
             'Session duration, in hours. Default: ' + options[:session_duration_hrs].to_s ) do |session_duration|
       options[:session_duration_hrs] = session_duration
     end
+    
+    opts.on('-s', '--summary', 'Summary mode. Default: ' + options[:summary].to_s ) do |summary|
```

Comment:
we discussed briefly and are thinking it's best not to assign the a-z single-letter flags yet to avoid potential future collisions with what we'd like to be the defaults for a given letter flag, so we may want to use just the fully-qualified named flags for now.